docs/server-workflow-search.md

```diff
@@ -4,57 +4,42 @@ title: Search for Temporal Server Workflows
 
-```sql
-WorkflowType = "main.Workflow" and Status != 0 and (StartTime > "2019-06-07T16:46:34-08:00" or CloseTime > "2019-06-07T16:46:34-08:00" order by StartTime desc)
-```
-
-## Memo vs Search Attributes
-
-Temporal offers two methods for creating Workflows with key-value pairs: memo and search attributes. Memo can only be provided on Workflow start. Also, memo data are not indexed, and are therefore not searchable. Memo data are visible when listing Workflows using the list APIs. Search attributes data are indexed so you can search Workflows by querying on these attributes. However, search attributes require the use of Elasticsearch.
-
-Memo and search attributes are available in the Go client in [StartWorkflowOptions](https://pkg.go.dev/go.temporal.io/sdk/internal#StartWorkflowOptions).
-
-```go
-type StartWorkflowOptions struct {
-    // ...
+To make this search functionality even better, the Temporal Server supports an integration with [Elasticsearch](https://www.elastic.co/elasticsearch/).
+By integrating with Elasticsearch, custom key-value search attributes can be added to Workflow metadata which can be referenced in SQL-like search queries.
 
-    // Memo - Optional non-indexed info that will be shown in list Workflow.
-    Memo map[string]interface{}
+Example:
 
-    // SearchAttributes - Optional indexed info that can be used in query of List/Scan/Count Workflow APIs (only
-    // supported when Temporal server is using Elasticsearch). The key and value type must be registered on Temporal server side.
-    // Use GetSearchAttributes API to get valid key and corresponding value type.
-    SearchAttributes map[string]interface{}
-}
+```sql
+WorkflowType = "main.Workflow" and Status != 0 and (StartTime > "2019-06-07T16:46:34-08:00" or CloseTime > "2019-06-07T16:46:34-08:00") order by StartTime desc
 ```
 
-In the Java client, the _WorkflowOptions.Builder_ has similar methods for [memo](https://javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/client/WorkflowOptions.Builder.html#setMemo-java.util.Map-) and search attributes.
-
-Some important distinctions between memo and search attributes:
-
-- Memo can support all data types because it is not indexed. Search attributes only support basic data types (including String, Int, Float, Bool, Datetime) because it is indexed by Elasticsearch.
-- Memo does not restrict on key names. Search attributes require that keys are allowlisted before using them because Elasticsearch has a limit on indexed keys.
-- Memo doesn't require Temporal clusters to depend on Elasticsearch while search attributes only works with Elasticsearch.
+Without Elasticsearch, search functionality is limited to listing Workflows by Id and type (open/closed), and search queries are run directly against the Temporal Server database.
+The Elasticsearch database takes on the search query request load, relieving potential performance issues that can occur.
```

Comment:
Technically ES is a database but nobody thinks of it that way. I would call it a "search engine" or just drop the qualifier altogether.